media/js/firefox/whatsnew/whatsnew-137-video-events.es6.js

```diff
@@ -0,0 +1,26 @@
+/*
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/.
+ */
+
+function recordInteraction(state) {
+    window.dataLayer.push({
+        event: 'video-interaction',
+        videoTitle: 'wnp-137',
```

Comment:
not sure if including a title attribute on the HTML video is useful, I left it out for simplicity but not a strong opinion